packages/eslint-config-sentry-app/index.js

```diff
@@ -1,14 +1,19 @@
 // Default: sentry app
 module.exports = {
   extends: [
+    'sentry-react',
+
+    // This prettier plugins need to be last so they can override plugin rules
+    // See https://github.com/prettier/eslint-config-prettier/blob/master/README.md#installation
+    // for plugin exclusions
     'prettier',
```

Comment:
It isn't last though.

Reply:
Comment is a bit wrong, should be "these" referring to all of the prettier ones